app/controllers/components/course/controller_component_host.rb

```diff
@@ -52,6 +53,25 @@ def gamified?
       def can_be_disabled?
         true
       end
+
+      # Returns a model which the current component can use to interface with its persisted
+      # settings.
+      #
+      # Example:
+      # If the component Course::FoobarComponent has settings, define a class
```

Comment:
As long as it gives an introduction to the new developer on what to do I think it works. What you have typed above seems ok.